sklearn/utils/sparsefuncs.py

```diff
@@ -361,7 +361,7 @@ def inplace_swap_row_csr(X, m, n):
 
 def inplace_swap_row(X, m, n):
     """
-    Swaps two rows of a CSC/CSR matrix in-place.
+    To swap two rows of a CSC/CSR matrix in-place.
```

Comment:
The title and the opening comment states that this updates `partial_dependence`, but `inplace_swap_row` is changed in this PR. Did you mean to update `partial_dependence`?

Reply:
Oh no, I was supposed to make another PR for `inplace_swap_row`  but it seems I've made a mistake. Is there any way to separate the 2 commits?

Reply:
You can run [git revert](https://git-scm.com/docs/git-revert) to revert the commit that changes `inplace_swap_row`.

Reply:
Thank you, I've fixed it now. I'll open another PR for `inplace_swap_row`.